### PR TITLE
[INLONG-5874][Agent] Use dataTime to report audit metrics

### LINKED ISSUE #5874: [Improve][Agent] Add msgTime attribute in PackProxyMessage
### Description

The parent issue is: https://github.com/apache/inlong/issues/5745

### InLong Component

InLong Agent

### Are you willing to submit PR?

- [X] Yes, I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### PR DESCRIPTION
**Pull request by pocozh**
### Prepare a Pull Request
*(Change the title refer to the following example)*

- Title Example: [INLONG-5874][Agent] Add msgTime attribute in PackProxyMessage

*(The following *XYZ* should be replaced by the actual [GitHub Issue](https://github.com/apache/inlong/issues) number)*

- Fixes #5874 

### Motivation

*Explain here the context, and why you're making that change. What is the problem you're trying to solve?*

### Modifications

*Describe the modifications you've done.*

### Verifying this change

*(Please pick either of the following options)*

- [ ] This change is a trivial rework/code cleanup without any test coverage.

- [ ] This change is already covered by existing tests, such as:
  *(please describe tests)*

- [ ] This change added tests and can be verified as follows:

  *(example:)*
  - *Added integration tests for end-to-end deployment with large payloads (10MB)*
  - *Extended integration test for recovery after broker failure*

### Documentation

  - Does this pull request introduce a new feature? (yes / no)
  - If yes, how is the feature documented? (not applicable / docs / JavaDocs / not documented)
  - If a feature is not applicable for documentation, explain why?
  - If a feature is not documented yet in this PR, please create a follow-up issue for adding the documentation

### CODE REVIEW

File: inlong-agent/agent-plugins/src/main/java/org/apache/inlong/agent/plugin/sinks/ProxySink.java [hunk 1]
```diff
@@ -104,11 +103,10 @@ public void write(Message message) {
                                 }
                                 // add message to package proxy
                                 packProxyMessage.addProxyMessage(proxyMessage);
-                                //
+                                // update msgTime
+                                packProxyMessage.updateMsgTime(System.currentTimeMillis());
```
Comment (luchunliang) [inline inlong-agent/agent-plugins/src/main/java/org/apache/inlong/agent/plugin/sinks/ProxySink.java:107]: It is better to use current time in Agent source.

Comment (luchunliang) [reply inlong-agent/agent-plugins/src/main/java/org/apache/inlong/agent/plugin/sinks/ProxySink.java:107]: Miss the modification about AUDIT_ID_AGENT_READ_SUCCESS.

Comment (pocozh) [reply inlong-agent/agent-plugins/src/main/java/org/apache/inlong/agent/plugin/sinks/ProxySink.java:107]: > It is better to use current time in Agent source.

In `agent-source`, messages are encapsulated piece by piece. But, they are packaged to be sent to `dataproxy` with only one  `msgTime`. So, is it better to use current time here?

**Review by healchow — APPROVED** [review]

**Review by luchunliang — APPROVED** [review]
LGTM

**Review by yunqingmoswu — APPROVED** [review]

**Review by gosonzhang — APPROVED** [review]

**Pull request merged**